P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AutoMapper;
+using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void RoleService_CreateData()
```

Comment:
@DanielAltamirano  According to the naming convention for tests, what do you think about `CreateRole_PersistsChange`?

Reply:
Sounds good!

Reply:
Great. Thank you!